app/frontend/ezra_init.js

```diff
@@ -51,9 +51,19 @@ $.create_xml_doc = function(string)
   return doc;
 }
 
+function earlyRestoreLocalizedString() {
+  const loadingStr = window.localStorage && window.localStorage.getItem('loading');
+  const loadingElement = document.querySelector('[i18n="general.loading"]');
+  if (loadingElement) {
+    loadingElement.textContent = loadingStr || 'Loading';
+  }
+}
+
 window.addEventListener('load', function() {
   console.log("load event fired!");
 
+  earlyRestoreLocalizedString();
```

Comment:
That's a good idea! I wanted to do something like that at some point ... Note that on Cordova there are currently some more strings that are not localized during startup. Since the startup takes longer on Cordova the individual steps are actual shown as second header underneath the Loading box (The element id is _loading-subtitle_). Eventually we would also need to add the other items here, but I can take care of that!